master/internal/config/oidc_config.go

```diff
@@ -17,6 +17,10 @@ type OIDCConfig struct {
 	AutoProvisionUsers          bool   `json:"auto_provision_users"`
 	GroupsAttributeName         string `json:"groups_attribute_name"`
 	DisplayNameAttributeName    string `json:"display_name_attribute_name"`
+	AgentUID                    int    `json:"agent_uid"`
+	AgentGID                    int    `json:"agent_gid"`
+	AgentUserName               string `json:"agent_user_name"`
+	AgentGroupName              string `json:"agent_group_name"`
```

Comment:
nit: Could we instead make these match `GroupAttributeName` & `DisplayNameAttributeName`, that way it's clear that these aren't the values for these fields, but instead the name of the attribute that can be passed in. Maybe something along the lines of `AgentUIDAttributeName`?

Reply:
Absolutely! I was wondering about the naming; what is the meaning of `AttributeName`? Is it kinda like a "key" in a key/value pair?

Reply:
Yep exactly!